src/classes/AFFL_MultiRecordTypeMapper.cls

```diff
@@ -93,31 +93,31 @@ public with sharing class AFFL_MultiRecordTypeMapper {
         // Map of Account RecordType Label To API
         accRecTypeLabelToAPI = getAccRecTypeLabelToAPIMap();
         
-        //List of primary affiliation lookup fields labels.
-        List<String> primaryAfflFieldLabels = accRecTypeToContactLabel.values();
+        //List of primary affiliation lookup fields labels/API names from settings.
+        List<String> primaryAfflFieldsFromsettings = accRecTypeToContactLabel.values();
```

Comment:
We might want to capitalize the 's' for settings.

Reply:
Thanks for catching that, fixed it!